src/main/plugin/dcat-ap/schema/dcat.xsd

```diff
@@ -327,6 +340,26 @@
     </xs:sequence>
   </xs:complexType>
 
+
+  <!-- dcat:DatasetSeries -->
+  <xs:complexType name="DatasetSeries_type">
```

Comment:
Note: I could not find an actual record describing a DatasetSeries so this is purely theoretical for now; open to any suggestions/help, thanks!